homeassistant/components/ps4/media_player.py

```diff
@@ -0,0 +1,390 @@
+"""
+Support for PlayStation 4 consoles.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/ps4/
+"""
+from datetime import timedelta
+import logging
+import socket
+import voluptuous as vol
+
+
+import homeassistant.helpers.config_validation as cv
+import homeassistant.util as util
+from homeassistant.components.media_player import (
+    ENTITY_IMAGE_URL, MEDIA_TYPE_MUSIC, MediaPlayerDevice,
+    SUPPORT_SELECT_SOURCE, SUPPORT_STOP, SUPPORT_TURN_OFF, SUPPORT_TURN_ON,
+)
+from homeassistant.components.ps4 import DOMAIN as PLATFORM
+from homeassistant.const import (
+    ATTR_ENTITY_ID, ATTR_COMMAND, CONF_HOST, CONF_NAME, CONF_REGION,
+    CONF_TOKEN, STATE_IDLE, STATE_OFF, STATE_PLAYING, STATE_UNKNOWN,
+)
+from homeassistant.util.json import load_json, save_json
+
+
+DEPENDENCIES = ['ps4']
+REQUIREMENTS = ['pyps4-homeassistant==0.2.5']
```

Comment:
You can remove the REQUIREMENTS here as it is already done in the component, don't forget to update `requirements_all.txt` after removing it.